Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/StakeHoldersComponent/table.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { Button } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Paper from "@material-ui/core/Paper";
+import TextField from "@material-ui/core/TextField";
+import "./table.css";
+
+const useStyles = makeStyles({
+  table: {
+    minWidth: 300,
+  },
+});
+
+/*
+function createData(issuename, score, maxpoints) {
+  return { issuename, score, maxpoints };
+}
+*/
+
+/*
+const rows = [
+  createData("Money", 1.0, 5.0),
+  createData("Life", 3.0, 5.0),
+  createData("Privacy", 4.0, 5.0),
+]; */
+
+export default function BasicTable(props) {
+  const classes = useStyles();
+
+  return (
+    <TableContainer component={Paper}>
+      <Table className={classes.table} aria-label="simple table">
+        <TableHead>
+          <TableRow>
+            <TableCell>Issue</TableCell>
+            <TableCell align="right">Score</TableCell>
+            <TableCell align="right">Max</TableCell>
+            <TableCell align="right"></TableCell>
```

Comment:
Why is 'max' and 'score' aligned right and 'issue' not? I would align them all to the left or align them center. 
![14](https://user-images.githubusercontent.com/51762139/96806242-88e1c980-13e1-11eb-963f-80f2ec08cf23.PNG)